crates/bitwarden-vault/src/cipher/ssh_key.rs

```diff
@@ -0,0 +1,43 @@
+use bitwarden_crypto::{
+    CryptoError, EncString, KeyDecryptable, KeyEncryptable, SymmetricCryptoKey,
+};
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Debug, JsonSchema, Clone)]
+#[serde(rename_all = "camelCase", deny_unknown_fields)]
+#[cfg_attr(feature = "uniffi", derive(uniffi::Record))]
+pub struct SshKey {
+    pub private_key: Option<EncString>,
+    pub public_key: Option<EncString>,
+    pub fingerprint: Option<EncString>,
+}
+
+#[derive(Serialize, Deserialize, Debug, JsonSchema, Clone)]
+#[serde(rename_all = "camelCase", deny_unknown_fields)]
+#[cfg_attr(feature = "uniffi", derive(uniffi::Record))]
+pub struct SshKeyView {
+    pub private_key: Option<String>,
```

Comment:
**note:** We should rarely need the private key, I wonder if we could decrypt it when we require it rather than always?

Reply:
What would be the recommendation here? Something like `decrypt_fido2_credentials` in bitwarden-uniffi/../ciphers.rs?

The cases where we need the ssh private key are:
- Revealing the private key in the view / edit components
- Exporting the private key
- Periodically syncing the private keys to the desktop_native module (though we might move this to sdk in the long run, not sure yet)

Reply:
I think we can keep this for a later refactor.